msm/src/column_env.rs

```diff
@@ -91,12 +92,33 @@ impl<'a, const N: usize, F: FftField> TColumnEnvironment<'a, F> for ColumnEnviro
         }
     }
 
-    fn column_domain(&self, _col: &Self::Column) -> Domain {
+    fn column_domain(&self, col: &Self::Column) -> Domain {
         // TODO FIXME check this is a tricky variable it should match the evalution in column
```

Comment:
This comment is removed in https://github.com/o1-labs/proof-systems/pull/2001. Sorry for the second PR. I realized it after creating the other PRs.